sdk/include/dxsdk/dmusici.h

```diff
@@ -21,11 +21,8 @@
 #ifndef __WINE_DMUSIC_PERFORMANCE_H
 #define __WINE_DMUSIC_PERFORMANCE_H
 
-#ifndef __WINESRC__
 #include <windows.h>
-#else
 #include <oleauto.h> /* VARIANT */
```

Comment:
NOTE: Wine headers don't have it.
https://github.com/wine-mirror/wine/blob/master/include/dmusici.h#L21

Reply:
Deleted.